src/sentry/api/endpoints/organization_user_teams.py

```diff
@@ -28,5 +28,13 @@ def get(self, request: Request, organization) -> Response:
             return Response(serialize(list(queryset), request.user, TeamWithProjectsSerializer()))
         else:
             return Response(
-                serialize(list(request.access.teams), request.user, TeamWithProjectsSerializer())
+                serialize(
+                    list(
+                        Team.objects.filter(
+                            id__in=[t.team.id for t in request.access.api_team_members]
```

Comment:
This is another good example -- because request.access is shared in both silo modes, it can't deal in concrete orm objects.  But this endpoint can, so it makes the translation.